src/plots/cartesian/tick_defaults.js

```diff
@@ -29,29 +29,27 @@ module.exports = function handleTickDefaults(containerIn, containerOut, coerce,
         delete containerOut.tickcolor;
     }
 
+    var showAttrDflt = getShowAttrDflt(containerIn);
+
+    var tickPrefix = coerce('tickprefix');
+    if(tickPrefix) coerce('showtickprefix', showAttrDflt);
+
+    var tickSuffix = coerce('ticksuffix');
+    if(tickSuffix) coerce('showticksuffix', showAttrDflt);
+
     var showTickLabels = coerce('showticklabels');
     if(showTickLabels) {
         Lib.coerceFont(coerce, 'tickfont', options.font || {});
         coerce('tickangle');
 
-        var showAttrDflt = getShowAttrDflt(containerIn);
-
-        if(axType !== 'category') {
-            var tickFormat = coerce('tickformat');
-            if(!options.noHover) coerce('hoverformat');
-
-            if(!tickFormat && axType !== 'date') {
-                coerce('showexponent', showAttrDflt);
-                coerce('exponentformat');
-            }
+        var tickFormat = coerce('tickformat');
+        if(!tickFormat && axType !== 'date') {
```

Comment:
Whoa good catch. When I pulled out `coerce('hoverformat')` I completely goofed on that. I'll add it into the `if` condition right now.